css/components/home.css

```diff
@@ -78,15 +92,19 @@
   padding: 10px;
   border-radius: 10px;
 }
+
 .shp-now-btn a {
   color: var(--text--primary--color);
 }
+
 .title {
   font-size: larger;
```

Comment:
you should use relative units for font size

Reply:
Updated